### PR TITLE
Implement user authentication

### PR DESCRIPTION
**Pull request by bluquar**
Followed [this guide](https://django-graphql-auth.readthedocs.io/en/latest/quickstart/#account-verification)

Create an account:

Screen Shot 2021-11-22 at 7 08 22 AM

Verify that account using the token sent via email:

Screen Shot 2021-11-22 at 7 09 50 AM

Update the account by setting my first name:

Screen Shot 2021-11-22 at 7 10 35 AM

Check that my first name was properly updated:

Screen Shot 2021-11-22 at 7 10 57 AM

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ django = "*"
 graphene-django = "*"
 django-cors-headers = "*"
 django-environ = "*"
+django-graphql-jwt = "1.7.0"
+django-graphql-auth = "*"
```
Comment (lowellbander) [inline Pipfile:12]: oh lovely, django ecosystem to the rescue?

File: community_aid_backend/schema.py [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
+class Query(UserQuery, MeQuery, graphene.ObjectType):
     all_aid_requests = graphene.List(AidRequestType)
 
     def resolve_all_aid_requests(root, info):
         return AidRequest.objects.all()
 
-schema = graphene.Schema(query=Query)
\ No newline at end of file
+
+class Mutation(AuthMutation, graphene.ObjectType):
```
Comment (lowellbander) [inline community_aid_backend/schema.py:34]: this is why's this here? does Graphene require the method exists, but we're choosing to implement a separate AuthMutation?

**Review by lowellbander — APPROVED** [review]
yas!! How're you liking this multi-repo approach? any friction points? anything that's easier?

File: Pipfile (hunk 1 above)
Comment (bluquar) [reply Pipfile:12]: yep!

File: community_aid_backend/schema.py (hunk 2 above)
Comment (bluquar) [reply community_aid_backend/schema.py:34]: this class is a misnomer (but a commonly used misnomer in all the docs/tutorials...)

it should be called "Mutations". it's a class where each property is a single mutation. the way you include mutations from libraries is through [multiple] inheritance, which is basically composition

so you can think of it like 

```
const mutations: GrapheneObjectType = {
  ...AuthMutations, 
  /* we will put our custom mutations here
}
```

**Comment by bluquar** [comment]
> yas!! How're you liking this multi-repo approach? any friction points? anything that's easier?

It's easier to have multiple PRs/branches out at once if they're across multiple repos since I don't have to deal with as much merging lol. 

Friction point is that I had to configure the security rules for requiring approvals to merge PRs separately in both repos

**Review by lowellbander — APPROVED** [review]

**Pull request merged**